test/integration/coredump/test_coredump.py

```diff
@@ -54,7 +65,7 @@ def apport_crash_to_coredump(crash):
         resource.setrlimit(resource.RLIMIT_CORE, (resource.RLIM_INFINITY, rlim_core_hard))
     # Crash tarantool.
     cmd = ["tarantool", "-e", "require('ffi').cast('char *', 0)[0] = 42"]
-    rc, output = run_command_and_get_output(cmd, cwd=coredump_tmpdir)
+    rc, output = run_command_and_get_output(cmd, cwd=coredump_dir, env=env)
```

Comment:
In this case, you will lose other environment settings that may be needed in some situations.
The right way would be to make a copy of the current environment and add new settings to it.
But, I suggest you make a copy of `tarantool` search logic from the `tt-ee` project, then you don't need to make such changes here, and nor add a new `fixture` `coredump_alt`.

Reply:
Adjusted to use executable path instead of changing environment.
Considering that we need 2 different coredumps in the scope of single test session I see no way to avoid using another fixture.